backend/src/utils/__mocks__/user.ts

```diff
@@ -0,0 +1,13 @@
+import { NextFunction, Request, Response } from 'express'
+
+export async function getUser(req: Request, _res: Response, next: NextFunction) {
+  console.log('auto mocked getUser')
```

Comment:
Either change these to use the `logger` or remove them?

Reply:
Oops, added debugging while development. Good spot